docs/topics/multiplatform-mobile/multiplatform-mobile-ktor-sqldelight.md

```diff
@@ -117,32 +115,12 @@ Both the `kotlinx.serialization` and SQLDelight libraries also require additiona
    ```kotlin
        plugins {
        // ...
-       kotlin("plugin.serialization") version "%kotlinVersion%"
-       id("com.squareup.sqldelight")
+       kotlin("plugin.serialization").version("%kotlinVersion%")
+       id("com.squareup.sqldelight").version("%sqlDelightVersion%")
    }
    ```
 
-3. Now go to the `build.gradle.kts` file in the project _root directory_ and specify the classpath for the plugin in the
-   build system dependencies:
-
-    ```kotlin
-    buildscript {
-        
-        dependencies {
-            // ...
-            classpath("com.squareup.sqldelight:gradle-plugin:%sqlDelightVersion%")
-        }
-    }
-    ```
-
-4. Finally, define the SQLDelight version in the `gradle.properties` file in the project _root directory_ to ensure that
-   the SQLDelight versions of the plugin and the libraries are the same:
-
-    ```none
-    sqlDelightVersion=%sqlDelightVersion%
```

Comment:
I couldn't find a reference to this property anywhere. Could it have been left by accident from an earlier tutorial revision or did I miss something?

Reply:
Yes, I guess it's something old. If it's not used in the sample project, we can remove it